content/client-server-api/modules/mentions.md

```diff
@@ -45,8 +76,7 @@ Clients should display mentions differently from other elements. For
 example, this may be done by changing the background color of the
 mention to indicate that it is different from a normal link.
 
-If the current user is mentioned in a message (either by a mention as
-defined in this module or by a push rule), the client should show that
+If the current user is mentioned in a message, the client should show that
 mention differently from other mentions, such as by using a red
 background color to signify to the user that they were mentioned.
```

Comment:
there is text below here that says group mentions look the same as room mentions. That is certainly no longer true. I feel like we should just remove the stuff about groups.